workers/unity/Assets/Fps/Scripts/GameLogic/Hud/ConnectionScreenController.cs

```diff
@@ -0,0 +1,85 @@
+using Improbable.Gdk.Core;
```

Comment:
Is this script required any more? Think it's from Martijn's gdk-for-unity connect screen

Reply:
Definitely good to be removed. Did any other scripts come over as part of that?

Reply:
I think so, will scour for them